prow/scripts/cluster-integration/README.md

```diff
@@ -68,3 +68,15 @@ This service account must have GCP permissions equivalent to the following GCP r
 - DNS Administrator (`roles/dns.admin`)
 - Service Account User (`roles/iam.serviceAccountUser`)
 - Storage Admin (`roles/storage.admin`)
+
+### Stackdriver Monitoring
+
+Long-running clusters on GKE use Stackdriver Monitoring to expose some performance metrics scraped by Kyma Prometheus instance.
+To send metrics to Stackdriver, the collector sidecar container is injected into Prometheus Pod.
+Add the container's environment variable to instruct the script to provision cluster with the Stackdriver collector.
+Long running clusters on GKE are using Stackdriver monitoring to expose some performance metrics scraped by Kyma Prometheus instance.
```

Comment:
This sentence is a repetition. We can delete it.

Reply:
done